game/scenery_group.py

```diff
@@ -40,8 +40,10 @@ def from_trigger_zones(trigger_zones: Iterable[TriggerZone]) -> List[SceneryGrou
 
         # For each objective definition.
         for zone_def in zone_definitions:
-
-            zone_def_radius = zone_def.radius
+            if isinstance(zone_def, TriggerZoneCircular):
+                zone_def_radius = zone_def.radius
+            else:
+                zone_def_radius = 1500  # use default radius for TriggerZoneCircular
```

Comment:
Either this, or raise an exception to indicate the given trigger zones aren't supported...

Reply:
@Snappy#7337 The latest pydcs version also updated trigger zones. The radius is no longer available on a regular 'TriggerZone', which causes a problem here. Looking at the file's history I see you added it 15 months ago, so I'm hoping you still remember something about it. 😅 

I try to cope with this by checking whether the object is an instance of 'TriggerZoneCircular', however I'm not sure how to deal with something else. Any change you remember what kind of trigger zones we're expecting here? If we're expecting a certain kind, we might as well raise an exception if it's not a 'TriggerZoneCircular'.